lib/msal-common/src/authority/Authority.ts

```diff
@@ -552,17 +552,25 @@ export class Authority {
                 : [];
             if (metadata.length === 0) {
                 // If no metadata is returned, authority is untrusted
+                this.logger.warning(`The cloud instance discovery network request's status code is: ${response.status}`);
```

Comment:
@Robbie-Microsoft , please make sure you log the full HTTP response, either here or at the network level as @tnorling  suggested.

Ultimately, go through all the logs that you just added and ensure we have a clear picture of what is happening. We already asked the affected partner to provide logs twice, let's not make him do it the thrid time. Please use log.verbose to log as much data is needed. The goal of verbose logs is to provide sufficient information to troubleshoot the issue, not to be succint or to provide good readability. That is secondary.